bin/triton-mysql.py

```diff
@@ -372,34 +371,43 @@ def on_change():
 
 def create_snapshot():
     log.debug('create_snapshot')
+    try:
+        backup_lock = open('/tmp/{}'.format(BACKUP_TTL_KEY), 'r+')
```

Comment:
this doesn't quite work as intended... we need the file in place first and that'll throw an IOError, which we're catching below for the lock. this needs to be in its own try/except block